pages/meander-rewriting.js

```diff
@@ -147,11 +147,12 @@ export default () =>
     <Clojure>
     {`
       (def find-x
-        (r/rewrite
-         (pred number? ?x) ?x
-         [?x] ?x
-         [?x _] ?x
-         [?x _ _] ?x))
+        (r/choice
```

Comment:
I'm not sure about this change but could not make it work without adding `r/choice`.